aspnet/fundamentals/environments.rst

```diff
@@ -72,6 +72,24 @@ For example, you can use the following code in you Configure method to setup env
 
 If the app is running in a ``Development`` environment, then it enables BrowserLink and development specific error pages (which typically should not be run in production). Otherwise, if the app is not running in a development environment, a standard error handling page is configured to be displayed in response to any unhandled exceptions.
 
+You may need to determine which content to send to the client at runtime, depending on the current environment. A common scenario when this happens is when a developer uses full scripts and style sheets for development and debugging vs. their minified counterparts for staging or production. You can do this by setting the names attribute of the ``<environment>`` Tag Helper that corresponds to the environment you want it to work in - ``Development``, ``Staging`` or ``Production``. 
+
+.. code-block:: html
+
+  <environment names="Development">
+    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
+    <link rel="stylesheet" href="~/css/site.css" />
+  </environment>
+  <environment names="Staging,Production">
+    <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.5/css/bootstrap.min.css"
+      asp-fallback-href="~/lib/bootstrap/dist/css/bootstrap.min.css"
+      asp-fallback-test-class="sr-only" asp-fallback-test-property="position"
+      asp-fallback-test-value="absolute" />
+    <link rel="stylesheet" href="~/css/site.min.css" asp-append-version="true" />
+  </environment>
+
+You can define environmental settings for use in Tag Helpers in the project’s debug properties or the ``launchSettings.json`` file.
```

Comment:
Not sure about focusing on the `launchSettings.json` file. The system has many ways to set the current environment and `launchSettings.json` is just as VS-specific as "the project’s debug properties".

Suggest instead mentioning two distinct options such as "set the `ASPNETCORE_ENVIRONMENT` environment variable at the command prompt (before starting a self-hosted site) or change the same variable in Visual Studio -- See the `Environment Variables` section of the project's Debug properties tab." (Note I'm not sure how to correctly reference that tab.)